helm/chaos-mesh/templates/chaos-daemon-daemonset.yaml

```diff
@@ -64,6 +64,9 @@ spec:
               value: {{ .Values.timezone | default "UTC" }}
             {{- end }}
           securityContext:
+            {{- if .Values.chaosDaemon.privileged }}
+              privileged: true
```

Comment:
Please remove 2 spaces here. 
I think `privileged: true` should keep the same indent level with `capabilities`.

Reply:
Fixed